composer.json

```
@@ -21,15 +21,17 @@
     },
     "require-dev": {
         "symfony/framework-bundle": "^3.0|^5.0",
-        "squizlabs/php_codesniffer": "^3.1",
+        "squizlabs/php_codesniffer": "^3.5",
         "phpunit/phpunit": "^8|^9",
         "symfony/twig-bundle": "^3.0|^4.0",
         "symfony/web-profiler-bundle": "^3.0|^4.0",
         "symfony/finder": "^3.0|^4.0",
         "symfony/class-loader": "^3.0",
         "symfony/templating": "^3.0|^4.0",
         "symfony/browser-kit": "^3.0|^4.0",
-        "symfony/form": "^3.0|^4.0"
+        "symfony/form": "^3.0|^4.0",
+        "phpstan/phpstan": "^0.12.14",
```

I would suggest to add the following packages also:
```suggestion
        "phpstan/phpstan": "^0.12.14",
        "phpstan/phpstan-strict-rules": "^0.12",
        "phpstan/phpstan-phpunit": "^0.12",
        "phpstan/phpstan-deprecation-rules": "^0.12",
```

Also, there is a dependency that allows an automatic installation of phpstan when extensions are declared:
`"phpstan/extension-installer": "^1.0.3"`
When used correctly, you don't need to maintain a lot the phpstan.neon (in my working project, I don't need phpstan.neon for instance, thanks to this package). It could be helpful.